app/frontend/src/assets/scss/style.scss

```diff
@@ -1,12 +1,28 @@
 /* Typography */
-body .v-application {
+.v-application {
   font-family: -apple-system, BlinkMacSystemFont, "BCSans", "Noto Sans", Verdana, Arial, sans-serif !important;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-  font-size: 1em;
-  line-height: 1.5em;
+  line-height: 1.5 !important;
 }
 
+h1 { font-size: 2.074em; /* (37.32px) */ }
+h2 { font-size: 1.728em; /* (31.10px) */ }
+h3 { font-size: 1.444em; /* (25.92px) */ }
+h4 { font-size: 1.2em; /* (21.60px) */ }
+
+h1, h2, h3, h4, h5 {
+  line-height: 1.2 !important;
```

Comment:
Would really not rather throw !important around so much. But as we've discussed Vuetify has a high specificity due to theme classes. I think we can fight this with our own theme down the line.

Reply:
yeah i agree.. i struggled to find a better way and then read it was common practice for overriding vuetify. i think it's ok as long as you can override this as well. one alternative would be to add a class to the body tag (or some other high-level html tag and then override the specificity(?) with a rule like: body.bcgov h1{...} not sure if that works.

Reply:
let me try and get rid of those important tags now

Reply:
Yeah I've had to add classes and do longer selectors to get the specificity up in the past, which is bad practice as well. No no winning strategy with Vuetify from what I've seen. I think it's ok to have the importants at this top level since it's defining it for the whole app, but again I think it's an issue we can kick down the road till we see what we can do with a theme.

Reply:
you were right.. there was no need for all those !importants